Recap-Project/recap-project-4/src/App.js

```diff
@@ -56,26 +82,42 @@ function App() {
     }
   })}
 
+  //Here I filter for all activities that have the key: isForGoodWeather === false
   const filteredBadWeatherActivities = activities.filter((activity) => activity.isForGoodWeather === isBadWeather)
 
+
   return (
     <div className="App">
-      <header className="App-header">
-      </header>
-      {/* passing the function handleAddActivity as a prop to the Form component */}
-      <Form onAddActivity={handleAddActivity} />
-
-      <h3>Bad Weather Activity</h3>
-      {/* Add a list element for good weather as well as for bad weather:
-      the activities which we defined in our state and filtered in GoodWeatherActivities as well as BadWeatherActivities are going to be displayed here separately*/}
-      <List 
-      activities={filteredBadWeatherActivities}
-            isGoodWeather={isBadWeather}
-      />
-      <h3>Good Weather Activity</h3>
-      <List activities={filteredGoodWeatherActivities(isGoodWeather)}
+      <>
+      {/* Here I use ternary operator to define if else. If the weather is good (line28 set as setWeather=data.condition) then the first condition after the "?" displays.
+      here in the list all the filtered Good Weather Activities will be displayed (line 76). Also in the isGoodWeather prop the condition is checked with the variable from line 50.
+      After the ":" the else case is displayed. Here the bad weather condition. */}
+        {weather ? (
+          <>
+            <h1>
+              {condition} {temperature}°C
+            </h1>
+            <p>Oh happy days! Weather is really good (could be ice cold though). Let's get rolling. You have the following things that you can do:</p>
+            <List activities={filteredGoodWeatherActivities(isGoodWeather)}
             isGoodWeather = {isGoodWeather}/>
+          </>
+        ) : (
+          <>
+            <h1>
+              {condition} {temperature}°C
+            </h1>
+            <p>Ahhh the weather today seems dreadful! But life is good so here are some things to do!</p>
+            <List 
+      activities={filteredBadWeatherActivities}
+            isGoodWeather={isBadWeather}/>
+          </>
+        )}
```

Comment:
You have a solution that works, but that could be more concise (works in combination with my comment from the previous PR):

```js
<>
            <h1>
              {condition} {temperature}°C
            </h1>
            <p>{weather ? "Oh happy days! Weather is really good (could be ice cold though). Let's get rolling. You have the following things that you can do:" . "Ahhh the weather today seems dreadful! But life is good so here are some things to do!"</p>
            <List activities={filteredActivities} isGoodWeather={isGoodWeather}/>
</>
```